### DISCUSSION TITLE
pg_hba.conf Configuration location

### DISCUSSION THREAD

**Discussion by dspeck1** [cloudnative-pg/cloudnative-pg#539] [Q&A] [answered]
A question not an issue.  I am troubleshooting configuring LDAP and want to examine the rendered pg_hba.conf.  I can't find it in the container filesystem.  I can see a /controller/manager that launches.  Is it possible to see the configuration files?

**Comment by sxd** [comment, marked as answer]
hi @dspeck1 
The pg_hba.conf file you can find it in the container if you get inside running the following command
```
cat $PGDATA/pg_hba.conf
```

**Reply by dspeck1** [reply]
Thanks!  Found it.
